wazo_call_logd/generator.py

```diff
@@ -16,8 +16,13 @@
 
 class CallLogsGenerator(object):
 
-    def __init__(self, cel_interpretors):
+    def __init__(self, confd, cel_interpretors):
+        self.confd = confd
         self._cel_interpretors = cel_interpretors
+        self._service_tenant_uuid = None
+
+    def set_token(self, token):
```

Comment:
```suggestion
    def set_default_tenant(self, token):
```
or something more explicit `set_default_fallback_tenant_uuid_from_service_call_logd_token_details_:trollface:()`

Reply:
I used set_default_tenant_uuid(()